rules/windows/builtin/system/win_susp_vuln_cve_2022_21919_or_cve_2021_34484.yml

```diff
@@ -0,0 +1,20 @@
+title: Suspicious usage of CVE 2022_21919 or CVE_2021_34484
```

Comment:
`title: Suspicious Usage of CVE 2022_21919 or CVE_2021_34484`
https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title